README.md

```diff
@@ -519,6 +520,23 @@ To run that image execute
 docker run --rm -t -i -p 8080:8080 moditect/vertx-helloworld
 ```
 
+## CDI
+
+The [POM file](integrationtest/cdi/pom.xml) in _integrationtest/cdi_
+shows another example. It uses Jakarta EE API (already modularized) adn adds module descriptors for
+[Weld](http://vertx.io) and its dependencies.
+
+Your can launch the build of the project and image with:
+
+`mvn clean package`
```

Comment:
Is `clean` really needed? Also, `mvn verify` would be the preferred way 😏

Reply:
Yes clean is needed because temp module folder should be cleaned before rebuilding the project. Otherwise add-module-info golf will fail on next build

Reply:
Huh, I thought we fixed that. I'll have a look as I believe this may be related to another bug recently opened.